Custom processor with two arguments (#186)
adamreisnz/replace-in-file: README.md

  processor: (input) => input.replace(/foo/g, 'bar'),
});
```

> It's also possible to use a custom processor with two arguments. In this case the first argument of the callback is still the content to be processed. The second one receives the file being processed.
> 

Reviewer: ```suggestion
The custom processor will receive the path of the file being processed as a second parameter:

```

```js
const results = replace.sync({
  files: 'path/to/files/*.html',

>   processor: (input, file) => input.replace(/foo/g, 'bar') + file,

Reviewer: ```suggestion
  processor: (input, file) => input.replace(/foo/g, file),
```
